mlserver/parallel/worker.py

```diff
@@ -52,11 +58,16 @@ def _executor(self):
         return self.__executor
 
     def run(self):
-        install_uvloop_event_loop()
-        configure_logger(self._settings)
-        configure_metrics(self._settings)
-        self._ignore_signals()
-        asyncio.run(self.coro_run())
+        ctx = nullcontext()
+        if self._env:
+            ctx = self._env
```

Comment:
nit: should this be inline?

Reply:
I've got mixed opinions about Python's inline syntax TBH. In this case, I felt it was clearer to have it separate.